<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatternEditorClass</class>
 <widget class="QWidget" name="PatternEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>709</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Page HTML source</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="urlLabel">
       <property name="text">
        <string>URL</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downloadButton">
       <property name="text">
        <string>Download</string>
       </property>
       <property name="icon">
        <iconset resource="../../siilihairesources.qrc">
         <normaloff>:/data/view-refresh.png</normaloff>:/data/view-refresh.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="viewInBrowserButton">
       <property name="text">
        <string>View in browser</string>
       </property>
       <property name="icon">
        <iconset resource="../../siilihairesources.qrc">
         <normaloff>:/data/emblem-web.png</normaloff>:/data/emblem-web.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="testPageSpanning">
       <property name="toolTip">
        <string>Run the test on as many pages as possible</string>
       </property>
       <property name="text">
        <string>Test page spanning</string>
       </property>
       <property name="icon">
        <iconset resource="../../siilihairesources.qrc">
         <normaloff>:/data/view-refresh.png</normaloff>:/data/view-refresh.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="sourceTextEdit">
     <property name="undoRedoEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="patternLabel">
     <property name="text">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans'; font-size:7pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Pattern: &lt;span style=&quot; font-weight:600;&quot;&gt;%a&lt;/span&gt;=id   &lt;span style=&quot; font-weight:600;&quot;&gt;%b&lt;/span&gt;=name   %c=last change&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="patternEdit"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTableWidget" name="resultsTable">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="errorLabel">
         <property name="text">
          <string>Possible errors</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="warningLabel">
         <property name="text">
          <string>Possible warnings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="countLabel">
         <property name="text">
          <string>No matches found</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../siilihairesources.qrc"/>
 </resources>
 <connections/>
</ui>
